src/shared/is_node.js

```diff
@@ -15,5 +15,10 @@
 /* globals module, process */
 
 module.exports = function isNodeJS() {
-  return typeof process === 'object' && process + '' === '[object process]';
+  // NW.js is a browser context, but copies some node objects.
+  // For more information see the 'Separate Context Mode' doc:
+  // http://docs.nwjs.io/en/latest/
```

Comment:
Why not link *directly* to the relevant page instead, to save people from having to search around for the referenced content?
Please just change this entire comment to something like this instead (note that URL in comments are except from the general line-length limit): 
```javascript
// NW.js is a browser context, but copies some Node.js objects; see
// http://docs.nwjs.io/en/latest/For%20Users/Advanced/JavaScript%20Contexts%20in%20NW.js/#separate-context-mode
```

Reply:
NW.js doc evolves so I prefered to not give a link that could be broken later, then harder to resolve than domain name + section name.